src/screens/CategoriesFilterScreen/component.js

```diff
@@ -34,9 +34,7 @@ class CategoriesFilterScreen extends PureComponent<Props> {
   };
 
   render() {
-    const { events, categories } = this.props;
-    const buttonLabel =
-      categories.size > 0 ? text.showEvents(events.length) : text.showAllEvents;
+    const { events, categories, applyButtonText } = this.props;
```

Comment:
I'm not 100% sure about this change. It was nice the fact that we could easily snapshot test this text based on the number of events passed. Can we avoid passing applyButtonText as a prop and instead call `text.filterPickerApply` here to calculate it?

Reply:
I ended up refactoring this to be consistent across all screens. Now we just expect the numberOfEvents to be passed in and we calculate the text, label and disabled from that.